lib/db.js

```diff
@@ -449,9 +449,9 @@ Store.prototype.update = function (query, object, fn) {
   debug('update - command', command);
 
   collection(this, function (err, col) {
-      col.update(query, command, {multi: multi}, function(err) {
+    col.update(query, command, { multi: multi }, function (err, r) {
       store.identify(query);
-      fn(err);
+      fn(err, r ? { n: r.result.n } : null);
```

Comment:
I am ok with n if for security reason.

Personally, I would like to use 'count' which is easy for documentation